libflux/src/core/scanner/scanner.h

```diff
@@ -17,6 +17,7 @@ enum TOK {
     OPTION                        = 11,
     BUILTIN                       = 12,
     TEST                          = 13,
+    TESTCASE                      = 56,
```

Comment:
Should we renumber everything? Otherwise it will be hard to know what the next max number to use is as you will have to manually scan the whole set.

Reply:
I didn't know if there was a requirement here to keep these enums this way, and I erred on the side of caution. If we don't do it now, maybe we can avoid it altogether when we remove the `testcase` keyword and just use `test`.